aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -142,4 +142,45 @@ Tensor flipud(const Tensor& self) {
   return self.flip({0});
 }
 
+Tensor moveaxis(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
```

Comment:
cc @mruberry for numpy compatibility -- the corresponding numpy operator is called `torch.moveaxis`, but I think to "torch-ify it" it should be called "torch.movedims" or "torch.movedim". What do you think?

Reply:
`torch.movedim` is definitely more PyTorch-like. We can add an alias for `moveaxis` later, too.